### PR TITLE
Fixed survey completions and enrollments form

### PR DESCRIPTION
**Pull request by adrianochuy**
Addesses #4

### CODE REVIEW

File: app/views/enrollments/_form.html.haml [hunk 1]
```diff
@@ -2,8 +2,9 @@
...
   = f.error_notification message: f.object.errors[:base].to_sentence if f.object.errors[:base].present?
   .form-inputs
-    = f.association :survey
-    = f.association :user
-    = f.input :status
-  .form-actions
+    = f.association :survey, collection: Survey.order(:survey_name), prompt: "Choose a Survey", label_method: :survey_name 
+    = f.association :user, collection: User.where.not(first_name: nil).order(:first_name), prompt: "Choose a Name", label_method: :full_name 
+    = f.input :status, collection: Enrollment.statuses.keys.map {|name| [name.titleize, name]}, prompt: "Status"
+  .form-actions
```
Comment (adrianochuy) [inline app/views/enrollments/_form.html.haml:8]: Fixed the select menus to populate the survey & user names & status

Comment (adrianochuy) [reply app/views/enrollments/_form.html.haml:8]: image

File: app/views/surveys/show.html.haml [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-%div
+%div 
   = link_to "Edit this survey", edit_survey_path(@survey)
   |
   = link_to "Back to surveys", surveys_path
   |
   = link_to "Enroll Survey", new_survey_enrollment_path(@survey), method: :post, class: "new"
   |
-  = link_to "Complete Survey", survey_completions_path(@survey), method: :post, class: "new"
+  = button_to "Complete Survey", survey_completions_path(@survey), method: :post, class: "new" unless @completed_survey
```
Comment (adrianochuy) [inline app/views/surveys/show.html.haml:10]: This fixes the POST action needed by your routes, it was doing a GET previously. also made it so it only shows the button if survey hasn't been completed

Comment (dowlingmi01) [reply app/views/surveys/show.html.haml:10]: Nice job! Thanks for fixing this.

**Review by dowlingmi01 — APPROVED** [review]
Nice job!

**Pull request merged**